src/style.css

```diff
@@ -1,3 +1,58 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 15px 50px;
+  height: 30px;
+}
+
+.nav__link {
+  display: inline-block;
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:first-child) {
+  padding-left: 20px;
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)